### DISCUSSION TITLE
What is non-conftest plugins?

### DISCUSSION THREAD

**Discussion by Tusenka** [pytest-dev/pytest#12998] [Q&A] [answered]
What are non-conftest plugins?

**Comment by webknjaz** [comment, marked as answer]
`pip install`ed ones.

**Comment by The-Compiler** [comment]
Just what it says - a [pytest plugin](https://docs.pytest.org/en/stable/how-to/plugins.html) that's installed as a Python package, as opposed to a  `conftest.py` file.
